<html>

<head>
<meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
<meta http-equiv="Content-Language" content="en-us">
<title>Interests</title>
<meta name="GENERATOR" content="Microsoft FrontPage 6.0">
<meta name="ProgId" content="FrontPage.Editor.Document">
<style>
<!--
div.Section1
	{page:Section1;}
li.MsoNormal
	{mso-style-parent:"";
	margin-bottom:.0001pt;
	font-size:12.0pt;
	font-family:"Times New Roman";
	margin-left:0in; margin-right:0in; margin-top:0in}
span.style51
	{color:#FFCC00}
h2
	{margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:3.0pt;
	margin-left:0in;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:Arial;
	font-weight:bold;
	font-style:italic}
.Section1 div #table1 tr td #table2 tr td #table3 tr td p strong {
	font-family: Cambria, Hoefler Text, Liberation Serif, Times, Times New Roman, serif;
}
.Section1 div #table1 tr td #table2 tr td #table3 tr td h3 strong u {
	font-family: Cambria, Hoefler Text, Liberation Serif, Times, Times New Roman, serif;
}
.Section1 div #table1 tr td #table2 tr td #table3 tr td h3 u b font {
	font-family: Cambria, Hoefler Text, Liberation Serif, Times, Times New Roman, serif;
}
-->
</style>
</head>

<body bgcolor="#33CCCC">

<table border="0" cellpadding="0" cellspacing="0" width="100%">
  <tr>
    <td width="150" colspan="2" align="center">&nbsp;</td>
    <td align="center" width="433">
    <font size="6" color="#FF0000">Major Contributions</font>
    </td>
  </tr>
  <tr>
    <td width="100%" colspan="3" style="border-bottom-style: none; border-bottom-width: medium">
    </td>
  </tr>
</table>
<table border="0" cellpadding="0" cellspacing="0" width="100%">
  <tr>
    <td style="border-bottom-style: solid; border-bottom-width: 1" align="right">
    <h4><!--webbot bot="Timestamp" S-Type="REGENERATED" S-Format="%m/%d/%y" startspan --><!--webbot bot="Timestamp" i-checksum="13364" endspan -->
    </h4>
    </td>
  </tr>
</table>
<table border="0" cellpadding="0" cellspacing="0" width="100%">
  <tr>
    <td valign="top" width="75" style="border-left-style:none; border-left-width:medium; border-right-style:solid; border-right-width:1; border-top-style:none; border-top-width:medium; border-bottom-style:none; border-bottom-width:medium">
      <p style="text-align:center" align="center"><a href="index.html" target="_top">Home</a></p>
      <p style="text-align:center" align="center"><a href="Research-interest.htm">Interests</a></p>
      <p style="text-align:center" align="center"><a href="Hu's Lab.htm">Research Lab</a></p>
      <p style="text-align:center" align="center"><a href="contribution.htm">Contributions</a></p>
      <p style="text-align:center" align="center"><a href="Publications.htm">Publications</a></p>
      <p style="text-align:center" align="center"><a href="Courses.htm">Courses</a></p>
      <p style="text-align:center" align="center"><a href="http://astro.temple.edu/%7Ewhu/favorite.htm">Favorites </a></p>
      <p style="text-align:center" align="center"><a href="aboutme-1.htm">About Me</a></p>
      <p style="text-align:center" align="center"><a href="feedback.htm">Feedback</a></p>
      <p align="center">&nbsp;</p>
&nbsp;<p>&nbsp;</p>
    </td>
    <td valign="top" width="5" style="border-right-style:none; border-right-width:medium; border-top-style:none; border-top-width:medium; border-bottom-style:none; border-bottom-width:medium">
    </td>
    <td valign="top" width="100%" height="100%" style="border-right-style:none; border-right-width:medium; border-top-style:none; border-top-width:medium; border-bottom-style:none; border-bottom-width:medium">
    <div class="Section1">
		<div align="center">
			<table style="width: 100%" cellSpacing="0" cellPadding="0" width="100%" border="0" id="table1">
				<tr style="HEIGHT: 130.5pt">
					<td style="PADDING-RIGHT: 0in; PADDING-LEFT: 0in; PADDING-BOTTOM: 0in; PADDING-TOP: 0in; HEIGHT: 130.5pt" vAlign="top">
					<table style="width: 100%" cellSpacing="0" cellPadding="0" width="100%" border="0" id="table2">
						<tr style="HEIGHT: 130.5pt">
							<td style="PADDING-RIGHT: 0in; PADDING-LEFT: 0in; PADDING-BOTTOM: 0in; WIDTH: 100%; PADDING-TOP: 0in; HEIGHT: 130.5pt" vAlign="top" width="100%">
							<table style="width: 100%" height="555" cellSpacing="4" cellPadding="0" align="left" border="0" id="table3">
								<tr style="HEIGHT: 259.5pt">
									<td style="PADDING-RIGHT: 1.5pt; PADDING-LEFT: 1.5pt; PADDING-BOTTOM: 1.5pt; PADDING-TOP: 1.5pt; HEIGHT: 259.5pt"><h3>The  challenges of discovery and translation inspire my biomedical research. I am  fascinated by the creative process of identifying novel molecules that regulate  signal transduction and gene expression. My achievements focus in 5 main areas:  NF<span style="font-family: Symbol">k</span>B  regulation, G protein signaling,  neurogenesis/neuronal polarization and CRISPR/Cas HIV cure. In my early studies I discovered  that TRAIL receptors induce apoptosis and activate NF<span style="font-family: Symbol">k</span>B and JNK, and discovered the potent  NF<span style="font-family: Symbol">k</span>B-activating roles  of the apoptosis-inducing proteins FADD, Casper and Caspase-8. My current and  planned researches are catalyzed by my discovery of 3 novel proteins (NIBP, TNAP  and TALL-1) that modulate NF<span style="font-family: Symbol">k</span>B signaling. NIBP enhances, while TNAP suppresses, NF<span style="font-family: Symbol">k</span>B activation by cytokines. NIBP regulates neuronal differentiation and  neurodevelopment, perhaps contributing to newly-identified neurodevelopmental  diseases (NIBP Syndrom). TALL-1 strongly modulates B-cell proliferation through non-classical  NF<span style="font-family: Symbol">k</span>B pathway, as  later discovered by others. NIMP, a novel mitochondrial  protein I discovered, interacts with nerve growth inhibitor Nogo. I tested astrocytic NF<span style="font-family: Symbol">k</span>B roles in neuropathogenesis by creating GFAP-dnI<span style="font-family: Symbol">k</span>B<span style="font-family: Symbol">a</span> transgenic mice, revealing reduced disease severity and improved  functional recovery in nerve injury models. I discovered that RGS4 is a novel  target gene regulated by NF<span style="font-family: Symbol">k</span>B signaling, that is differentially modified by MAPKs and PI3K/Akt/GSK3&beta;  signaling, and that RGS4 mRNA stability is regulated by HuR.  I identified  a new function of NF<span style="font-family: Symbol">k</span>B signaling that initiates  early differentiation of neural stem cells, by repressing C/EBP&beta;. These  findings have advanced the fields of neuroinflammation and neurogenesis, and  positioned me as a visionary leader to continue driving discovery of novel  roles and applications of NIBP/NF<span style="font-family: Symbol">k</span>B signaling in the nervous system. Before being independent faculty in 2008, my  research was focused on spinal cord injury (SCI), gene regulation (NIBP and  RGS4) and signal transduction (NFkB and G protein  signaling). My PhD thesis identified the critical role of nitric oxide in  dynorphin-induced spinal neurotoxicity. My postdoctoral training identified a novel function of the  apoptosis-inducing proteins FADD, Casper and Caspase-8 to activate NFkB signaling. Then I discovered 3 novel proteins (NIBP, TNAP and TALL-1)  that modulate NFkB signaling and one new mitochondrial protein NIMP that may regulate  neural regeneration. At the early stage of faculty, I discovered that RGS4 is a  novel NFkB target gene that is also modulated by MAPKs and PI3K/GSK3&beta; pathways. I  identified a new function of NFkB signaling to initiate early differentiation of neural stem cells (<strong>NSC)</strong>.  Since joining Temple University in 2008 as an independent faculty, I have demonstrated  an outstanding track record of extramural research funding (10 R01s and 2 R21s) and Department of Defense in the past 15 years with total cost $26 million, focusing  on targeted gene therapy (AAV, lentivirus-like particles and nanoparticles), CRISPR/Cas genome editing, HIV/NeuroHIV cure,  neurodevelopmental disorders and gastrointestinal diseases.  My new lentivirus-like particle technology that can cross BBB and  deliver editors to various neural cells received NIH TARGETED Challenge Prize  (Phase I) in 2023. My research has generated more than 130 publications in high-impact  journals including PNAS, Mol Therapy, Cell Reports, JEM, JCI, Stem Cells, JBC,  etc. I had 8 patents. One of them was exclusively licensed to Excision BioTherapeutics, currently at phase 1/2 clinical trial  in HIV patients. Another patent on novel coding-motif Exin21 to boost  antibody/vaccine/virus production is exclusively licensed to ExinBioPharma. His  papers have been cited by &gt;7000 times with h-index 42. </h3>
									  <h3><strong><u>I. CRISPR/Cas9-mediated HIV excision and reactivation </u></strong><strong> </strong></h3>
                                      <p>For a cure of HIV-1 patients, there is an urgent need of developing a  novel strategy that is capable of eradicating the integrated viral genome or  eliminate HIV latent cells. We utilized CRISPR/Cas9 technology to specifically  cut the long-term repeat (LTR) regions at multiple sites within HIV-1 genome.  We identified highly specific targets within LTR U3 region that inactivate  viral gene expression and replication in latently infected microglial,  promonocytic, and T cells [Huge global media response with the Altmetric score  881:&nbsp;<a href="https://pnas.altmetric.com/details/2526844">https://pnas.altmetric.com/details/2526844</a>. Ranked <a href="https://news.temple.edu/news/2015-01-08/temple-aids-discovery-among-top-100-science-stories-2014">2014 discovery at #86</a> among 100 top stories by &ldquo;Discover&rdquo; magazine. Cas9/gRNAs caused neither  genotoxicity nor off-target editing to the host cells(Hu et al., 2014).  Furthermore, a combination of sgRNAs targeting LTRs and the viral structural  genes provides a more efficient means of HIV-1 eradication in cultured cells  and prevents viral escape(Kaminski et al., 2016b; Kaminski et al.,  2016c; Yin et al., 2016). AAV9-mediated  saCas9/duplex sgRNA excised the integrated HIV-1 genome in HIV-1 transgenic  mice and rat(Kaminski et al., 2016a).&nbsp;<a href="http://www.biodiscover.com/news/celebrity/654850.html">AAVDJ8-mediated saCas9/quadruplex sgRNA excised HIV-1 genome in  humanized BLT mice and EcoHIV-infected mice</a>(Yin et al.,  2017). Our results suggest that Cas9/gRNA multiple cutting provides a  specific, efficacious prophylactic and therapeutic approach against AIDS. We  also identified a hotspot near to the enhancer of HIV LTR promoter targeted by  dCas9-SAM/msgRNA system to induce robust and persistent reactivation of HIV  latency leading to an explosive and rapid suicide death of latent cells(Zhang et al., 2015b). This  targeted latency-reversing approach adds a new therapeutic to the &ldquo;shock and  kill&rdquo; strategy to cure HIV/AIDS.</p>
                                      <p><br>
                                        <strong><u>II. Pseudovirus for SARS-CoV2 variants and therapeutic  exploration.</u></strong> <br>
                                      During the pandemic, my lab is actively collaborating with Dr. Wenzhe  Ho&rsquo;s lab to explore the cellular/molecular mechanisms and potential  therapeutics for the infection of the SARS-CoV2 and its variants. My lab  generated a series of plasmids bearing all mutations of the spike protein (S)  corresponding to the original SARS-CoV2 and its emerging variants. These  constructs were essential for timely generating pseudovirus of SAR-CoV2  variants for urgent needs in several collaborative labs. Earlier studies demonstrated  that green tea beverage (GTB) or its major ingredient, epigallocatechin gallate  (EGCG), are highly effective in inhibiting SARS-CoV2 infection. The potential  mechanism involves the interference with the engagement of the receptor binding  domain of the viral spikes to angiotensin-converting enzyme 2 (ACE2) receptor  of the host cells(Liu et al., 2021a). As  the COVID-19 vaccines first became available, we collected sera from vaccinated  individuals to assess the efficacy of vaccine-elicited antibodies against  SARS-CoV2 and its variants(Liu et al., 2021b). Our  studies found that vaccinations with either Pfizer or Moderna vaccine were able  to produce effective antibodies against spike proteins of both SARS-CoV2 WT and  the variants. There was an overall positive correlation between serum IgG  levels and ID50&nbsp;titers for not only SARS-CoV2 WT but also the variants,  suggesting that the level of IgG titer may be a correlate of immunity. Thus, it  is necessary to longitudinally monitor specific serum IgG levels for evaluating  the protective efficacy of vaccines against SARS-CoV2 and its new variants.  More and more evidence&nbsp;identified neurological symptoms and  neuropsychiatric disorders related to COVID-19 (Neuro-SARS2). Accumulated  robust evidence indicates that Neuro-SARS2 may play an important role in  aggravating the disease severity and mortality. Understanding the  neuropathogenesis and cellular mechanisms underlying Neuro-SARS2 is crucial for  both basic research and clinical practice to establish effective strategies for  early detection/diagnosis, prevention, and treatment. We performed a  comprehensive analysis on Neuro-SARS2 and proposed iPS-derived cerebral  organoids for mechanistic studies using CRIPSR/Cas13d RNA editor. </p>
                                      <p><br>
                                        <strong><u>III. Neurogenic deficits in HIV-associated neurocognitive  disorders (HAND)</u></strong> <br>
                                      Even in the antiretroviral treatment (ART)  era, HIV-1-infected patients suffer from milder forms of HIV-1-associated  neurocognitive disorders (HAND). While the viral proteins Tat and gp120 have  been shown to individually inhibit the proliferation and neural differentiation  of neural stem cells (NSCs), no studies have characterized the effects of the  combined viral proteins on adult neurogenesis. The HIV-1 Tg26 transgenic mouse  model resembles clinical patients in the post-ART era, who lack active viral  replication, but suffer from continuous stress from the combined viral  proteins. Using quantitative RT-PCR analysis, we validated varying copy numbers  of partial gag (p17), tat (unspliced and spliced variants), env (gp120), vpu,  and nef transcripts in the neurogenic zones of Tg26 mice. <em>In vitro</em> stemness and lineage differentiation assays as well as <em>in vivo</em> hippocampal neurogenic lineage  analysis revealed that HIV-1 Tg26 mice have early and late-stage neurogenic deficits  (Putatunda et al., 2018). The general  behavior screening and fear conditioning assessments revealed that Tg26 mice  exhibited no significant differences in general behavioral function, contextual  fear conditioning, or cued fear conditioning responses when compared to their  wild-type (WT) littermates, regardless of sex. However, Barnes maze testing  revealed significantly impaired short and long-term spatial memory in males,  and impaired spatial learning abilities and short-term spatial memory in  females. Compared to WT mice, both male and female Tg26 mice had fewer NSCs and  neuroblasts in their hippocampi. Male Tg26 mice had a more robust reduction of  the quiescent NSC pool than female Tg26 mice. While female WT mice had a higher  number of neural progenitor cells (NPCs) than male WT mice, only female Tg26 mice  exhibited a robust reduction in the number of NPCs. These results suggest that  male and female Tg26 mice manifest differential deficits in cognitive  functioning and hippocampal neurogenesis. This study emphasizes the importance  of understanding sex related differences in HAND pathology, which would aid in  designing more optimized therapeutic regimens for the treatment of HAND  (Putatunda et al., 2019).</p>
<h3><u><b><font size="6" color="#FF00FF">IV. Transcription Factor 4 (TCF4) resitricts neurite branching and synapse formation</font></b></u></h3>
<p>Proneural proteins of the class I/II  basic-helix-loop-helix (bHLH) family are highly conserved transcription  factors. Class I bHLH proteins are expressed in&nbsp;a broad number of tissues  during development, whereas class II bHLH protein expression is more tissue  restricted. Our understanding of the function of class I/II bHLH transcription  factors in both invertebrate and vertebrate neurobiology is largely focused on  their function as regulators of neurogenesis. TCF4 mutations have been reliably  identified in genome-wide association studies as a susceptibility risk factor  for <a href="http://topics.sciencedirect.com/topics/page/Schizophrenia">schizophrenia</a> and have also been associated with Pitt-Hopkins syndrome, Fuchs&rsquo; endothelial  corneal dystrophy, and primary sclerosing. It is unknown whether these diseases  are due to defects in neurogenesis, in mature differentiated cells, or both.  Our collaborative studies show that the class I bHLH proteins Daughterless and Tcf4 are expressed in postmitotic&nbsp;neurons in  Drosophila melanogaster and mice, respectively, where they function to restrict  neurite branching and synapse formation by repressing the expression of the  cell adhesion molecule Neurexin</p>
<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: 0in"><u><b><span lang="EN-US"><font size="6" color="#FF00FF">V. NF-<font face="Symbol">k</font>B 
									  signaling</font></span></b></u>: In my early studies I  discovered that TRAIL receptors induce apoptosis and activate NFkB and JNK(Hu, Johnson et al. 1999), and apoptosis-inducing proteins FADD, Casper  and Caspase-8 also activate NFkB signaling (Hu, Johnson et al. 2000). My current  researches are  catalyzed by the discovery of 3 novel proteins (NIBP, TNAP and TALL-1) that  modulate NFkB signaling. NIBP enhances(Hu, Pendergast et al. 2005), while TNAP suppresses(Hu, Mo et al. 2004), NFkB activation by cytokines.  NIBP regulates neuronal differentiation and neurodevelopment, perhaps  contributing to newly-identified neurodevelopmental diseases including NIBP  syndrome. TALL-1 strongly modulates B-cell proliferation through non-classical  NFkB pathway. These early contributions have over 1100  citations. </p>
<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: 0in">
							  <b>1<span lang="EN-US">. Identification of 
									three novel components of the NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US">B 
							  signaling pathway (TNAP, NIBP, TALL-1)</span></b></p>
									<p class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in"><b><i><span lang="EN-US">&nbsp;1) 
									<a href="file:///E:/My%20Files/HuWH'sPub/TNAP-JBC.pdf">TNAP</a>. </span></i></b></p>
									<p class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in"><span lang="EN-US">Using yeast two-hybrid system with NIK (<b><u>N</u></b>F</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US">B-<b><u>i</u></b>nducing
									<b><u>k</u></b>inase) as bait to screen cDNA 
									libraries from brain, spleen, leukocyte and 
									HEK293T cells, several known proteins such 
									as TRAF3, PAX6, CDC23 and proteasome subunit 
									PSMA3, and two novel proteins, TNAP and NIBP; 
									the latter were characterized in detail and 
									shown to interact with NIK.&nbsp; TNAP (<b><u>T</u></b>RAFs 
									and <b><u>N</u></b>IK-<b><u>A</u></b>ssociated
									<b><u>P</u></b>rotein) specifically 
									inhibited TNF-</span><span lang="EN-US" style="font-family: Symbol">a</span><span lang="EN-US"> 
									and IL-1</span><span lang="EN-US" style="font-family: Symbol">b</span><span lang="EN-US">-induced 
									NF-&#954;B activation by interacting with NIK and 
									TRAF2/3, and suppressing NIK kinase 
									activity. Thus, TNAP regulated both 
									classical (I</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US">B</span><span lang="EN-US" style="font-family: Symbol">a</span><span lang="EN-US"> 
									phosphorylation and degradation) 
									and non-classical (p100 processing to p52) pathways of NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US">B 
									activation. TNAP also suppressed TNF</span><span lang="EN-US" style="font-family: Symbol">a</span><span lang="EN-US">-induced 
									and NIK-mediated Ser<sup>536</sup> 
									phosphorylation of p65. </span></p>
									<p class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph">
									<b><i><span lang="EN-US">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
									2)&nbsp; 
									<a href="file:///E:/My%20Files/HuWH'sPub/NIBP-JBC.pdf">NIBP</a>. &nbsp;</span></i></b></p>
									<p style="text-align: justify; text-justify: inter-ideograph; text-indent: 27.0pt; margin-left: 0in; margin-right: 0in; margin-top: 0in; margin-bottom: .0001pt">
									<b>
									<span lang="EN-US" style="font-family: Times New Roman">
									NIBP <u>(N</u></span></b><span lang="EN-US" style="font-family: Times New Roman">IK 
									and <b><u>I</u></b>KK</span><span lang="EN-US" style="font-family: Symbol">b</span><span lang="EN-US" style="font-family: Times New Roman">
									<b><u>B</u></b>inding <b><u>P</u></b>rotein) 
									was mainly expressed in brain, muscle, heart 
									and kidney, and moderately expressed in 
									immune tissues such as spleen, thymus and 
									peripheral blood leukocytes, where NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US" style="font-family: Times New Roman">B 
									was known to modulate immune function. NIBP 
									physically interacted with NIK, IKK</span><span lang="EN-US" style="font-family: Symbol">b</span><span lang="EN-US" style="font-family: Times New Roman">, 
									but not IKK</span><span lang="EN-US" style="font-family: Symbol">a</span><span lang="EN-US" style="font-family: Times New Roman"> 
									or IKK</span><span lang="EN-US" style="font-family: Symbol">g</span><span lang="EN-US" style="font-family: Times New Roman">.&nbsp; 
									NIBP over-expression potentiated TNF</span><span lang="EN-US" style="font-family: Symbol">a</span><span lang="EN-US" style="font-family: Times New Roman"> 
									and IL-1</span><span lang="EN-US" style="font-family: Symbol">b</span><span lang="EN-US" style="font-family: Times New Roman">-induced 
									NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US" style="font-family: Times New Roman">B 
									activation through
									increased phosphorylation of the IKK complex 
									and its downstream I</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US" style="font-family: Times New Roman">B</span><span lang="EN-US" style="font-family: Symbol">a</span><span lang="EN-US" style="font-family: Times New Roman"> 
									and p65 substrates. Knockdown of NIBP 
									expression by lentiviral vector-mediated 
									small interfering RNA reduced TNF</span><span lang="EN-US" style="font-family: Symbol">a</span><span lang="EN-US" style="font-family: Times New Roman">-induced 
									NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US" style="font-family: Times New Roman">B 
									activation, prevented NGF-induced neuronal 
									differentiation and decreased the expression 
									of NF</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US" style="font-family: Times New Roman">B-dependent 
									gene, Bcl-xL, in PC12 cells. These data 
									demonstrated that NIBP, by interacting with 
									NIK and IKK</span><span lang="EN-US" style="font-family: Symbol">b</span><span lang="EN-US" style="font-family: Times New Roman">, 
									was a novel enhancer of&nbsp; cytokine-induced 
									NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US" style="font-family: Times New Roman">B 
									signaling.</span></p>
									<p class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in"><b><i><span lang="EN-US">3)&nbsp; 
									<a href="file:///E:/My%20Files/HuWH'sPub/TALL-JLB.pdf">TALL-1</a>.</span></i></b></p>
									<p class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in"><b><i><span lang="EN-US">&nbsp;</span></i></b><span lang="EN-US">Using 
									amino acid homology analysis, another novel 
									member of TNF ligand family was identified 
									and designated TALL-1 (for <b><u>T</u></b>NF- 
									and <b><u>A</u></b>poL-related <b><u>L</u></b>eukocyte 
									expressed <b>L</b>igand <b><u>1</u></b>).&nbsp; 
									Simultaneously, TALL-1 was discovered by 
									others and designated BAFF, BLyS, and zTNF4. 
									TALL-1 was a potent modulator of B-cell 
									proliferation via its receptors BCMA (B cell 
									maturation antigen) and TACI (transmembrane 
									activator and CAML-interactor), and was 
									expressed by monocytes/macrophages and 
									dendritic cells.</span></p>
									<p class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">&nbsp;</p>
									<h2 style="page-break-after: auto">
									<span lang="EN-US" style="font-size: 12.0pt; font-family: Times New Roman; font-style: normal">
									2.&nbsp; Characterization of NF</span><span lang="EN-US" style="font-size: 12.0pt; font-family: Symbol; font-style: normal">k</span><span lang="EN-US" style="font-size: 12.0pt; font-family: Times New Roman; font-style: normal">B 
									signaling pathways</span></h2>
									<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: .25in">
									<span lang="EN-US">As shown above, TNAP 
									suppressed and NIBP enhanced 
									cytokine-induced NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US">B 
									activation. Our earlier studies demonstrated 
									that receptors for TRAIL (<b><u>T</u></b>NF-<b><u>r</u></b>elated
									<b><u>a</u></b>poptosis-<b><u>i</u></b>nducing
									<b><u>l</u></b>igand) induced apoptosis, and 
									NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US">B 
									and JNK activation through distinct 
									signaling pathways. The apoptosis-inducing 
									adaptors FADD (Fas-associated via Death 
									Domain), Casper and Caspase-8 potently 
									activated NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US">B, 
									whereas “activated” Caspase-8 blocked NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US">B 
									activation by inactivating NIK. These data 
                                    were corroborated by several other groups.</span></p>
									<p class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in"><span lang="EN-US">&nbsp;Neurons and their neighboring cells 
									employ the NF-&#954;B pathway for distinctive 
									functions, ranging from development to 
									neuronal plasticity and coordination of 
									cellular responses to injury.&nbsp; As part of 
									our studies,&nbsp; I&#954;B</span><span lang="EN-US" style="font-family: Symbol">a</span><span lang="EN-US">-dominant 
									mutant transgenic mice were generated which 
									stably expressed mutant I&#954;B</span><span lang="EN-US" style="font-family: Symbol">a</span><span lang="EN-US"> 
									under the control of an astrocyte-specific 
									promoter, GFAP (glial fibrillary acidic 
									Protein) or a neuron-specific promoter, 
									synapsin.&nbsp; Selective inactivation of 
									astroglial NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US">B 
									in transgenic mice led to marked improvement 
									in function 8 weeks after contusive spinal 
									cord injury.&nbsp; The mice showed reduced 
									expression of pro-inflammatory chemokines 
									and cytokines, such as CXCL10, CCL2, and TGF-</span><span lang="EN-US" style="font-family: Symbol">b</span><span lang="EN-US">2.&nbsp; 
									Inactivation of astroglial NF-</span><span lang="EN-US" style="font-family: Symbol">k</span><span lang="EN-US">B 
									in transgenic mice led to a significant 
									deficit in learning and memory. </span></p>
									<p class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">&nbsp;</p>
<h2><span style="page-break-after: auto"><span lang="EN-US" style="font-size: 12.0pt; font-family: Times New Roman; font-style: normal">3.&nbsp; NF</span><span lang="EN-US" style="font-size: 12.0pt; font-family: Symbol; font-style: normal">k</span><span lang="EN-US" style="font-size: 12.0pt; font-family: Times New Roman; font-style: normal">B 
									signaling pathway initiates early neurogenesis</span></span></h2>
									<p><span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">&nbsp;Both <span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">NF&kappa;B</span> signaling and neurogenesis are currently two hot  topics in modern biomedical science. The epigenic and transcriptional  regulation in embryonic and adult neurogenesis is drawing wide attention. NF&kappa;B  signaling regulates neurite outgrowth and neural plasticity, as well as the  proliferation/apoptosis and terminal differentiation of neural stem cells  (NSCs). Early neurogenesis from NSCs produces identical progeny through  symmetric division and committed daughter neural progenitor cells (NPCs)  through asymmetric division. Whether <span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">NF&kappa;B</span> signaling regulates  initial differentiation and asymmetric division of NSCs and the factors  involved still remains largely unknown. In this study, we employed multiple  systems to demonstrate for the first time that <span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">NF&kappa;B</span> signaling initiates the differentiation  of the quiescent NSCs at the very early stage of both embryonic and adult  neurogenesis. The canonical IKK2/I&kappa;B&alpha;/p65 pathway is activated during the  initial stage of neural differentiation. NSC-specific inhibition of NF&kappa;B in  transgenic mice causes an accumulation of NSCs. Inhibition of NF&kappa;B signaling <em>in vitro</em> blocks differentiation and  asymmetric division and maintains NSCs in an undifferentiated state. This key finding  may explain a previous observation that the proliferation of NSCs but not  intermediate NPCs is significantly reduced by stress-induced <span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">NF&kappa;B</span> activation in the adult brain (Koo et al., 2010).  Our finding also supports a recent report showing that PEDF enhances NSC  self-renewal through the promotion of p65 nucleo-cytoplasmic export (Andreu-Agullo et al.,  2009).  In addition, we found that C/EBP&beta; is one of the key effectors of <span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">NF&kappa;B</span> signaling for the modulation of early asymmetric division and differentiation  of NSCs. Our findings will advance our understanding of the molecular mechanisms  not only for neural development and endogenous neurogenesis under normal  conditions, but also for the inducible regulation of neurogenesis after  injuries or diseases in the nervous system. Enrichment of NSCs by <span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">NF&kappa;B</span> inhibition may provide an invaluable tool to expand neurospheres. </span></p>
									<p>&nbsp; </p>
									<h2><span lang="EN-US" style="font-size: 12.0pt; font-family: Times New Roman; font-style: normal">4. NF<span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">&kappa;</span>B and MAPK signaling regulates RGS4</span></h2>
									<p class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">Regulator of G-protein Signaling 4 (RGS4) regulates  the strength and duration of Gai/G&alpha;q signaling and plays an important role in  regulating smooth muscle contraction/relaxation, cardiac development, neural  plasticity and psychiatric disorder. However, the underlying regulatory  mechanisms remain elusive. Our  studies showed that pro-inflammatory  cytokine IL-1b  up-regulates Rgs4 expression in rabbit colonic smooth muscle cells through the  canonical IKK2/I<span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">&kappa;</span>B<span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">&alpha;</span> pathway of NF<span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">&kappa;</span>B  activation as well as ERK1/2 and p38 MAPK pathways. This up-regulation of Rgs4  is negatively regulated by the activation of PI3K/Akt/GSK3<span class="MsoNormal" style="text-align:justify;text-justify:inter-ideograph;
text-indent:.25in">&beta;</span> pathway and MEKK1-MKK4-JNK-AP1 pathway. RGS4 mRNA stability is  regulated by HuR. GATA-6 transcriptional factor is essential for RGS4 transcriptional  regulation. These findings provide a novel and comprehensive understanding of  the signaling regulation on RGS4 expression. The positive and negative  regulatory mechanisms of RGS4 expression reflect an intricate and delicate  system for gene regulation. Such orchestral regulation may aid in maintaining  the transient function of RGS4 for smooth muscle contraction/relaxation as well  as cardiovascular and neuronal functions. </p>
									<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: 0in">
									<u><font size="6" color="#FF00FF"><b>VI<span lang="EN-US">. 
									Secondary spinal cord injury and neural 
									growth inhibition</span></b></font></u></p>
									<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: .25in">
									Around twenty years ago, the 
									pathophysiologic mechanism of the secondary 
									spinal cord injury was a hot spot in the 
									field of central nervous system injury. 
									Using evoked potentials to evaluate spinal 
									cord function and biomicrosphere technique 
									to measure spinal cord blood flow, my 
									studies demonstrated that ischemia in the 
									white matter is closely correlated with 
									spinal cord dysfunction after balloon 
									compressive injury in dogs. My further works 
									together with my colleagues corroborated the 
									important role of excitotoxicity and 
									lipoxidation in secondary spinal cord 
									injury. These studies were awarded a Second 
									Prize of Military Science and Technology 
									Achievement in 1998. </p>
									<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: .25in">
									During my PhD study, I continued the 
									research on the excitotoxic mechanism of 
									spinal cord injury using pharmacological 
									animal model, focusing on the role of 
									NMDA-Ca2+-NOS/NO pathway. NO was a &quot;molecule 
									of the year 1992 in science&quot;. My research 
									demonstrated for the first time that neurotoxic dose of dynorphin (an endogenous 
									opioid peptide) induces high expression of 
									both neuronal and inducible nitric oxide 
									synthases (nNOS and iNOS) in the spinal cord 
									of rats. Selective inhibition of either nNOS 
									or iNOS is neuroprotective while 
									non-selective NOS inhibition aggravates 
									dynorphin-induced spinal cord injury. NMDA 
									receptor functional activity is 
									significantly elevated in the ventral spinal 
									cord of rats with dynorphin spinal 
									neurotoxicity. In cultured spinal cord 
									neurons, high concentration of dynorphin 
									produces persistent calcium overload, which 
									is antagonized by pretreatment with both 
									NMDA receptor antagonist and kappa opioid 
									receptor antagonist. These data were granted 
									a Second Prize of Beijing Science and 
									Technology Achievement in 2000.</p>
									<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: .25in">
									Excitatory amino acids transpo<span lang="EN-US">rters 
									(EAAT) are essential to prevent 
									excitotoxicity and to terminate 
									glutamatergic neurotransmission. During my 
									postdoctoral training in Miami Project to 
									Cure Paralysis, I observed, unexpectedly, 
									that EAAT4 immunoreactivity is highly 
									enriched in the spinal cord. Further studies 
									demonstrated that EAAT4 is expressed in the 
									astrocytes of spinal cord at both protein 
									and mRNA levels. This astrocytic 
									localization of EAAT4 may reveal some new 
									function of EAAT4 in the spinal cord. </span>
									</p>
									<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: .25in">
									<span lang="EN-US">To develop a better 
									understanding of the mechanisms responsible 
									for the functions of Nogo, an important 
									myelin-derived nerve growth inhibitor after 
									spinal cord injury, I performed a yeast 
									two-hybrid screen of human brain cDNA 
									library using Nogo-66 as bait.&nbsp; A novel 
									mitochondrial protein designated NIMP (for
									<b>N</b>ogo-<b>I</b>nteracting <b>M</b>itochondrial
									<b>P</b>rotein) is highly conserved and 
									ubiquitously expressed in neurons and 
									astrocytes. Other two-mitochondrial proteins 
									UQCRC1/2 also interact with Nogo, indicating 
									that Nogo may affect mitochondrial 
									functions. </span></p>
									<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: 0in">
									<b>
									<span lang="EN-US" style="text-transform:uppercase">
									&nbsp;</span></b></p>
									<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: 0in">
									<u><font size="6" color="#FF00FF"><b>VII<span lang="EN-US">. 
									Receptor mechanism for S1P signaling</span></b></font></u></p>
									<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: .25in">
									<span lang="EN-US">Sphingosine-1-</span>phosphate<span lang="EN-US"> 
									(S1P) regulates diverse biological processes 
									through five receptor types, S1P<sub>1-5</sub>. 
									S1P induces an initial Ca<sup>2+</sup>)-dependent 
									contraction followed by a sustained Ca<sup>2+</sup>-independent, 
									RhoA-mediated contraction in rabbit gastric 
									smooth muscle cells. The cells coexpress S1P<sub>1</sub> 
									and S1P<sub>2</sub> receptors, but the 
									signaling pathways initiated by each 
									receptor type and the involvement of one or 
									both receptors in contraction are not known. 
									Lentiviral vector-mediated siRNA silencing 
									of S1P<sub>1</sub> receptors abolished 
									S1P-stimulated activation of G</span><span lang="EN-US" style="font-family: Symbol">a</span><sub><span lang="EN-US">i3</span></sub><span lang="EN-US"> 
									and partially inhibited activation of G</span><span lang="EN-US" style="font-family: Symbol">a</span><sub><span lang="EN-US">i1</span></sub><span lang="EN-US">, 
									whereas silencing of S1P<sub>2</sub> 
									receptors abolished activation of G</span><span lang="EN-US" style="font-family: Symbol">a</span><sub><span lang="EN-US">q</span></sub><span lang="EN-US">, 
									G</span><span lang="EN-US" style="font-family: Symbol">a</span><sub><span lang="EN-US">13</span></sub><span lang="EN-US">, 
									and G</span><span lang="EN-US" style="font-family: Symbol">a</span><sub><span lang="EN-US">i2</span></sub><span lang="EN-US"> 
									and partially inhibited activation of G</span><span lang="EN-US" style="font-family: Symbol">a</span><sub><span lang="EN-US">i1</span></sub><span lang="EN-US">. 
									Silencing of S1P<sub>2</sub> but not S1P<sub>1</sub>&nbsp;receptors 
									suppressed S1P-stimulated PLC-</span><span lang="EN-US" style="font-family: Symbol">b</span><span lang="EN-US"> 
									and Rho kinase activities, implying that 
									both signaling pathways were mediated by S1P<sub>2</sub> 
									receptors. The results obtained by receptor 
									silencing were corroborated by receptor 
									inactivation. The selective S1P<sub>1</sub> 
									receptor agonist SEW2871 did not stimulate 
									PLC-</span><span lang="EN-US" style="font-family: Symbol">b</span><span lang="EN-US"> 
									or Rho kinase activity or induce initial and 
									sustained contraction; when this agonist was 
									used to protect S1P<sub>1</sub>&nbsp;receptors so 
									as to enable chemical inactivation of S1P<sub>2</sub> 
									receptors, S1P did not elicit contraction, 
									confirming that initial and sustained 
									contraction was mediated by S1P<sub>2</sub> 
									receptors. Thus S1P<sub>1</sub> and S1P<sub>2</sub> 
									receptors are coupled to distinct 
									complements of G proteins. Only S1P<sub>2</sub> 
									receptors activate PLC-</span><span lang="EN-US" style="font-family: Symbol">b</span><span lang="EN-US"> 
									and Rho kinase and mediate initial and 
									sustained contraction.</span></p>
									<p class="MsoNormal" style="text-align: justify; text-justify: inter-ideograph; text-indent: .25in">
									<br>
&nbsp;</td>
								</tr>
							</table>
							</td>
						</tr>
					</table>
					</td>
				</tr>
			</table>
		</div>
	</div>
    </td>
  </tr>
</table>
<p class="MsoNormal" style="text-align: center; text-indent: .25in"><a href="https://www.vcu.edu/">VCU</a></p>
<DIV align=left>
<TABLE height=8 cellSpacing=0 cellPadding=0 width=920 id="table4" border="1"><!-- MSTableType="layout" -->
  <tr>
	<td width="918" height="6"></td>
	</tr>
</TABLE></DIV>
<p><i><small>This site was last updated
<!--webbot bot="Timestamp" S-Type="EDITED" S-Format="%m/%d/%y" startspan -->08/28/2024&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </small></i></p>

</body>

</html>